tests/rbd/rbd_delete_protected_snapshot_krbd.py

```diff
@@ -0,0 +1,97 @@
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.krbd_io_handler import krbd_io_handler
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def rbd_delete_protect_snap(rbd, pool_type, **kw):
+    """
+    This module on krbd client - Performs deletion of protected snapshot on a RBD image
```

Comment:
please update the doc string , Since this is one of the step in validation.

Reply:
done